### PR TITLE
💄 Add styled welcome screen

### PR DESCRIPTION
**Pull request by scopsy**

### CODE REVIEW

**Comment by scopsy** [comment]
@djabarovgeorge Also refactored the code a bit to get rid of the query params altogether and just used your rendering engine. Which was a great idea to build it like this. Love that.

File: packages/cli/src/server/http-server.ts [hunk 1]
```diff
@@ -30,6 +31,12 @@ export class HttpServer {
     this.server.listen(await getServerPort(), SERVER_HOST);
   }
 
+  public getAddress() {
+    const response = this.server.address() as AddressInfo;
+
+    return `http://localhost:${response.port}`;
```
Comment (djabarovgeorge) [inline packages/cli/src/server/http-server.ts:37]:
```suggestion
    return `http://${SERVER_HOST}:${response.port}`;
```

**Review by djabarovgeorge — COMMENTED** [review]
LOVE THE NEW INTRO! 
It looks so much nicer :)

**Comment by djabarovgeorge** [comment]
> @djabarovgeorge Also refactored the code a bit to get rid of the query params altogether and just used your rendering engine. Which was a great idea to build it like this. Love that.

There are quite large computational complications in use of the engine however in our case it is quite negligible. Glad you liked it :)

**Review by djabarovgeorge — APPROVED** [review]

**Pull request merged**